plenum/server/consensus/ordering_service.py

```diff
@@ -2334,6 +2333,11 @@ def _process_pre_prepare_from_old_view(self, pp):
 
         return PROCESS, None
 
+    def _reordered_in_new_view(self):
+        self._stasher.process_all_stashed(STASH_VIEW_3PC)
+        # TODO: Why do we call it "reordered" despite that we only _started_ reordering old batches?
```

Comment:
Agree with the comment. I was under the impressions that we send it after we ordered (re-ordered) till prepared certificate, but it looks like we send it when just going to do the re-ordering.

Reply:
Yet I think that it is right place to call `process_all_stashed`, so probably it is just a matter of renaming. However I'm not so sure about monitor reset...